java/Ice/secure/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) ZeroC, Inc.
+
+plugins {
+    // Register the Ice Builder plugin without applying it here.
+    // The plugin will be applied to all subprojects in the `subprojects` block below.
+    id("com.zeroc.ice.slice-tools") version "3.8.0-+" apply false
+}
+
+subprojects {
+    // Apply the application plugin to enable Java application support in subprojects.
+    apply(plugin = "application")
+
+    // Apply the Gradle Slice Tools for Ice and Java plugin to enable Slice compilation in all subprojects.
```

Comment:
"Gradle Slice Tools for Ice and Java plugin" is a bit long. Why not simply the "Slice tools plugin"? We don't refer to the application plugin above as the "Gradle application for Java plugin"!